### PR TITLE
docs: split up 'creating a fuel dapp' guide into multiple pages

### LINKED ISSUE #2087: Docs: Consider splitting `Creating a Fuel dApp` page
Content could become more accessible with sub-sections.

Example:
 - Initializing the project
 - Directory Structure
 - Dev Environment Setup
 - Adding Decrement Functionality
   - Modifying the Sway Contract
   - Modifying the Frontend
 - Deploying to testnet?

### PR DESCRIPTION
**Pull request by Dhaiwat10**
Closes #2087

### CODE REVIEW

File: apps/docs/src/guide/creating-a-fuel-dapp/adding-decrement-functionality.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
+# Adding Decrement Functionality
+
+To add decrement functionality to our counter, we will have to do two things: 1. Add a `decrement_counter` function to our Sway contract, and 2. Modify the `./src/pages/index.tsx` file to add a button that calls this function.
```
Comment (maschad) [inline apps/docs/src/guide/creating-a-fuel-dapp/adding-decrement-functionality.md:3]: plus forgive my grammar nits but _To add decrement_ is a bit ironic 😄

```suggestion
In order to introduce decrement functionality to our counter, we will have to do two things: 1. Introduce a `decrement_counter` function to our Sway contract, and 2. Modify the `./src/pages/index.tsx` file to introduce a button that calls this function.
```

File: apps/docs/src/guide/creating-a-fuel-dapp/adding-decrement-functionality.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+## 2. Modifying the Frontend
+
+We will now add a new button to the frontend that will call the `decrement_counter` function when clicked. To do this, we will modify the `./src/pages/index.tsx` file.
+
+First, we will add a function called `onDecrementPressed` similar to the `onIncrementPressed` function:
+
+<<< @/../../create-fuels-counter-guide/src/pages/index.tsx#create-fuels-counter-guide-on-decrement-react-function{ts:line-numbers}
+
+Second, we will add a new button to the UI that will call the `onDecrementPressed` function when clicked:
```
Comment (maschad) [inline apps/docs/src/guide/creating-a-fuel-dapp/adding-decrement-functionality.md:29]:
```suggestion
Secondly, we will add a new button to the UI that will call the `onDecrementPressed` function when clicked:
```

**Review by maschad — CHANGES_REQUESTED** [review]
Great work! Looks like CI docs are complaining about the new folder structure though

File: apps/docs/src/guide/creating-a-fuel-dapp/adding-decrement-functionality.md [hunk 3]
```diff
@@ -0,0 +1,45 @@
+# Adding Decrement Functionality
+
+To add decrement functionality to our counter, we will have to do two things: 1. Add a `decrement_counter` function to our Sway contract, and 2. Modify the `./src/pages/index.tsx` file to add a button that calls this function.
+
+## 1. Modifying the Sway Contract
+
+To add a `decrement_counter` function to our Sway contract, we will modify the `./sway-programs/contract/src/main.sw` file.
```
Comment (maschad) [inline apps/docs/src/guide/creating-a-fuel-dapp/adding-decrement-functionality.md:7]: See https://github.com/FuelLabs/fuels-ts/pull/2134#discussion_r1575541584

**Comment by Dhaiwat10** [comment]
Closing as not planned after #2135

**Comment by arboleya** [comment]
I still think it makes sense to split this, but that will require a slight change in the menu order.

We can revisit this as we go.

**Pull request closed without merging**